public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -67,12 +66,10 @@ export class PanelQueryRunner {
   private subscription?: Unsubscribable;
   private lastResult?: PanelData;
   private dataConfigSource: DataConfigSource;
-  private dataSupport?: PanelPluginDataSupport;
 
   constructor(dataConfigSource: DataConfigSource) {
     this.subject = new ReplaySubject(1);
     this.dataConfigSource = dataConfigSource;
-    this.dataSupport = this.dataConfigSource.dataSupport;
```

Comment:
I removed this as a property on dataConfigSource (PanelModel) as it was sometimes undefined when created before the plugin was loaded.